scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,212 @@
+
+\subsection{Namelist for the CCPP Suite Simulator}
+
+The CSS has its own namelist, \execout{ccpp\_suite\_sim\_nml}, that needs to be added to the physics namelists used by the SCM.
+
+\lstinputlisting[
+                 basicstyle=\scriptsize\ttfamily,
+                 label=lst_css_nml,
+                 caption=Example namelist for CCPP Suite Simulator.
+                 ]{./css_nml.txt}
+
+\begin{itemize}
+\item \execout{suite\_sim\_file}: Input file with simulated data tendencies (See \ref{section:Creating_Custom_Data_for_Simulator} for how to create input file from SCM output).
+\item \execout{nprc\_sim}: Number of physical processes in the input data.
+\item \execout{prc\_XYZ\_cfg}: Configuration for physical process XYZ.
+\begin{itemize}
+\item 0 - Active scheme; 1 - Use data
+\item 0 - Process-split scheme; 1 - Time-split scheme
+\item Index for scheme order (1 - \execout{nprc\_sim})
+\end{itemize}
+\end{itemize}
+
+For example, in \ref{lst_css_nml}, there are two active schemes, longwave and shortwave radiaiton, and five simulated schemes: PBL, gravity-wave drag, deep/shallow convection, and cloud microphysics. The radiation, gravity-wave drag and PBL schemes are all process-split, whereas convection and cloud microphysics are time-split. 
+
+\subsection{Creating Custom Data for Simulator}
+\label{section:Creating_Custom_Data_for_Simulator}
+
+Navigate to \execout{ccpp-scm/scm/etc/scripts/ccpp\_scheme\_sim}
+
+Provided with the SCM are scripts to generate data for the suite simulator using output from a previous SCM run. The first script, \execout{create\_1D\_CSSdata.py}, extracts the physics tendencies from a user specified time interval, which are used for constant forcing in the suite simulator. The other script, \execout{create\_2D\_CSSdata.py}, creates a two-dimensional forcing dataset. The suite simulator interpoaltes these forcings in time.
+
+\begin{enumerate}
+\item Run the SCM, using TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/bin
+./run_scm.py -c twpice -s SCM_GFS_v17_p8
+\end{lstlisting}
+\item Create 2D forcing data for the CSS, using SCM output from TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/etc/scripts/ccpp_scheme_sim
+./create_2D_CSSdata.py --cases twpice --suites SCM_GFS_v17_p8
+\end{lstlisting}
+\item Create constant forcing data for the CSS, using SCM output, at forecast time 3600s, from TWPICE case with \execout{GFS\_v17\_p8} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/etc/scripts/ccpp_scheme_sim
+./create_1D_CSSdata.py --cases twpice --suites SCM_GFS_v17_p8 --time 3600
+\end{lstlisting}
+\end{enumerate}
+
+The data file will be written to \execout{ccpp-scm/scm/etc/scripts/ccpp\_scheme\_sim/} with the following format, \execout{data\_scheme\_sim\_DIM.CASES.SUITES.nc}.
+
+\subsection{Example 1: Suite with Active Radiation}
```

Comment:
The CSS data was created with v17_p8, but this example is using v16 - not sure if it matters

Reply:
Good catch. I will use 2d data for one example (radiation), and 1d for the other (cloud mP)